static/src/javascripts/projects/common/modules/article/spacefinder.js

```diff
@@ -175,7 +175,7 @@ define([
         var body;
 
         rules = rules || defaultRules;
-        body = rules.bodySelector ? document.querySelector(rules.bodySelector) : document;
+        body = (rules.bodySelector && document.querySelector(rules.bodySelector)) || document;
```

Comment:
oh interesting, I rarely come across such syntactic fine-tuning: is it more efficient? you could do without the brackets but it would hinder readability.